query-connector/src/app/(pages)/queryBuilding/components/SelectionTable.tsx

```diff
@@ -31,11 +32,10 @@ export const ConceptTypeSelectionTable: React.FC<
 > = ({ vsTypeLevelOptions, handleVsTypeLevelUpdate, searchFilter }) => {
   const [expanded, setExpandedGroup] = useState<string>("");
   const [valueSetDisplay, setValueSetDisplay] =
-    useState<ConceptTypeToDibbsVsMap>(vsTypeLevelOptions);
```

Comment:
this state was getting initialized badly so that when there were changes to `vsTypeLevelOptions` (ie when the condition tabs were being toggled through) this value wasn't getting updated / the accordions weren't getting regenerated. Setting it to an empty object and then updating the value as a `useEffect` on 62 is correctly flushing the UI with new data